cmd/crank/xpkg/push.go

```diff
@@ -42,10 +42,32 @@ const DefaultRegistry = "xpkg.upbound.io"
 
 // pushCmd pushes a package.
 type pushCmd struct {
+	// Arguments.
+	Image string `arg:"" help:"The OCI image name to push the package to."`
```

Comment:
Now looks like this:

```
$ go run cmd/crank/main.go xpkg push --help
Usage: crossplane xpkg push <package>

Push a package to a package registry.

Packages can be pushed to any OCI registry. Packages are pushed to the
xpkg.upbound.io registry by default. A package's OCI tag must be a semantic
version.

Examples:

    # Push a package.
    crossplane xpkg push -f function-example.xpkg crossplane/function-example:v1.0.0

    # Push the xpkg file in the current directory to a different registry.
    crossplane xpkg push index.docker.io/crossplane/function-example:v1.0.0

Arguments:
  <package>    Where to push the package.
```

It intentionally leans on the examples to demonstrate what `<package>` is, because it's hard to describe succinctly (technically it's an OCI repository and server tag, optionally prefixed by a registry address). I'm also avoiding "package name" to avoid confusion with e.g. kind: Provider `metadata.name`.